test/e2e/mock-e2e.js

```diff
@@ -878,6 +903,28 @@ async function setupMocking(
       };
     });
 
+  // Snaps: Execution environment html
+  await server
+    .forGet(/^https:\/\/execution\.metamask\.io\/iframe\/[^/]+\/index\.html$/u)
```

Comment:
no matter which version we use (as it can change over time), this mock will be always applied